CAIPs/caip-x.md

```diff
@@ -0,0 +1,339 @@
+  "result": {
+    "scopes": {
+      "eip155:1": {
+        "methods": ["eth_signTransaction"],
+        "notifications": ["accountsChanged"],
+        "accounts": ["eip155:1:0xabc123"]
+      },
+      "eip155:137": {
+        "methods": ["eth_sendTransaction"],
+        "notifications": ["chainChanged"],
+        "accounts": ["eip155:137:0xdef456"]
+      }
+    }
+  }
+}
+```
+
+**Explanation:**
+
+- The `provider_getSession` method returns the current permissions for the session. It lists all scopes along with their methods, notifications, and accounts.
+
+### Events
+
+#### `providerAuthorizationChanged`
+
+This event is published by the wallet to notify the caller/dapp of updates to the session authorization scopes. The event payload indicates how the scopes have changed, showing additions and removals in the permissions.
+
+**Event Payload:**
+
+- `sessionId` (string, optional): The session identifier.
+- `scopes` (object, required): An object containing the updated session scopes, formatted according to CAIP-217.
+
+**Initial Session Scopes:**
+
+```json
+{
+  "eip155:1": {
+    "methods": ["eth_signTransaction"],
+    "notifications": ["accountsChanged"],
+    "accounts": ["eip155:1:0xabc123"]
+  },
+  "eip155:137": {
+    "methods": ["eth_sendTransaction"],
+    "notifications": ["chainChanged"],
+    "accounts": ["eip155:137:0xdef456"]
+  }
+}
+```
+
+**Example Event:**
+
+```json
+{
+  "event": "providerAuthorizationChanged",
+  "data": {
+    "scopes": {
+      "eip155:1": {
+        "methods": ["eth_signTransaction", "eth_sendTransaction"],
+        "notifications": ["accountsChanged"],
+        "accounts": ["eip155:1:0xabc123"]
+      },
+      "eip155:137": {
+        "methods": ["eth_sendTransaction"],
+        "notifications": [],
+        "accounts": ["eip155:137:0xdef456"]
+      }
+    }
+  }
+}
+```
+
+This event indicates how the scopes have changed by comparing the updated scopes with the initial session scopes. In the example, the method `eth_sendTransaction` was added to `eip155:1`, and the notification `chainChanged` was removed from `eip155:137`.
+
+### Optional SessionIds
+
+The `sessionId` parameter in the new lifecycle methods is optional. When not provided, the methods will operate on the current active session. This approach allows for more flexible session management without the overhead of tracking session identifiers.
+
+## Security Considerations
+
+The introduction of these lifecycle methods must ensure that only authorized parties can modify the permissions of a session. Proper authentication and authorization mechanisms must be in place to prevent unauthorized access or modifications.
+
+## Privacy Considerations
+
+Managing permissions within an existing session reduces the need to create multiple session identifiers, which can help minimize the exposure of session-related metadata. However, care must be taken to handle these methods in a way that does not inadvertently leak sensitive information.
```

Comment:
> Managing permissions within an existing session reduces the need to create multiple session identifiers
Here I feel like "session" would be more precisely worded "connection", and if it were, "multiple identifiers" wouldn't make any sense.  to put it another way, status quo is that wallet/provider persists the config of each connection and gives that config a connection-identifier which it only shares with the counterparty to that connection, so they aren't "multiple" unless provider/wallet is inadequately partioning its memory and storage between counterparties? 

I don't follow the bit about leaking information, since the current paradigm is to drop any messages not containing the counterparty-specific, opaque token and starting over at the first sign of mismatch between what you're persisting and what your counterparty is supposed to be persisting...